### PR TITLE
Query multiple table

### PR DESCRIPTION
**Pull request by Dadadon**
- Created a table named vets
- There is a many-to-many relationship between the tables species and vets: a vet can specialize in multiple species, and a species can have multiple vets specialized in it. Created a "join table" called specializations to handle this relationship
- There is a many-to-many relationship between the tables animals and vets: an animal can visit multiple vets and one vet can be visited by multiple animals. Created a "join table" called visits to handle this relationship, it should also keep track of the date of the visit
- Inserted data for vets
- Inserted data for specializations
- Inserted data for visits

**Write queries to answer the following**

- Who was the last animal seen by William Tatcher?
- How many different animals did Stephanie Mendez see?
- List all vets and their specialties, including vets with no specialties.
- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
- What animal has the most visits to vets?
- Who was Maisy Smith's first visit?
- Details for most recent visit: animal information, vet information, and date of visit.
- How many visits were with a vet that did not specialize in that animal's species?
- What specialty should Maisy Smith consider getting? Look for the species she gets the most.

Screenshot 2022-12-17 080508
Screenshot 2022-12-17 080528
Screenshot 2022-12-17 083206

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -87,4 +87,38 @@ SELECT * FROM animals anims JOIN owners own ON anims.owner_id=own.id JOIN specie
...
+
+-- Details for most recent visit: animal information, vet information, and date of visit.
+SELECT animals.id AS animal_id, animals.name AS animal_name, vets.id AS vet_id, vets.name AS vet_name, visit_date FROM visits JOIN animals ON animals.id = visits.animal_id JOIN vets ON visits.vets_id = vets.id ORDER BY visit_date LIMIT 1;
+
+
+-- How many visits were with a vet that did not specialize in that animal's species?
+SELECT count(animals.name) FROM visits JOIN animals ON animals.id = visits.animal_id JOIN specializations ON specializations.vets_id = visits.vets_id where animals.species_id <> specializations.species_id;
+
+-- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+SELECT species.name FROM visits JOIN vets ON visits.vets_id = vets.id JOIN animals on animals.id = visits.animal_id JOIN species on species.id = animals.species_id WHERE vets.name = 'Maisy Smith' GROUP BY animal_id, species.name ORDER BY COUNT(animal_id) DESC LIMIT 1;
```
Comment (alexander16108) [inline queries.sql:122]: Hi there 👋🏾 , sorry but u need to go a little bit further before I can finally approve your request, please add screenshots of all your queries `I can see some were added but not all` so we can make sure you were able to successfully implement the required data into your database. 👋🏾  bye bye for now.
cheers and happy holiday  🥂

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Dadadon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Dadadon) [reply queries.sql:122]: @alexander16108 It was excluded by mistake. I just update the pr description good you have a look?

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (alexander16108) [reply queries.sql:122]: it looks great now weldone

**Pull request merged**